src/2018/transitioning/modules/path-clarity.md

```diff
@@ -10,11 +10,23 @@ As such, the 2018 edition of Rust introduces a few new module system
 features, but they end up *simplifying* the module system, to make it more
 clear as to what is going on.
 
+Note: During the 2018 edition preview, there are two variants of the module
+system under consideration, the "absolute use paths" variant and the "relative
+paths" variant. Most of these changes apply to both variants; the two variant
+sections call out the differences between the two. We would encourage testing
+of the new "relative paths" variant introduced in edition preview 2. The
+release of the 2018 edition will use one of these two variants.
+
 Here's a brief summary:
 
 * `extern crate` is no longer needed
-* Absolute paths begin with a crate name, where the keyword `crate`
-  refers to the current crate.
+* The `crate` keyword refers to the current crate.
+* Relative paths variant: Paths work uniformly in both `use` statements and in
```

Comment:
I think `use` should be referred to as an item and not a statement, https://doc.rust-lang.org/reference/items/use-declarations.html